gym_lowcostrobot/envs/lift_cube_env.py

```diff
@@ -238,14 +265,21 @@ def step(self, action):
         cube_pos = self.data.qpos[:3]
         cube_z = cube_pos[2]
         ee_id = self.model.body("moving_side").id
-        ee_pos = self.data.geom_xpos[ee_id]
+        ee_pos = self.data.xpos[ee_id]
         ee_to_cube = np.linalg.norm(ee_pos - cube_pos)
+        print(f"Cube position: {cube_pos}, EE position: {ee_pos}, Distance: {ee_to_cube}")
```

Comment:
Drop the print (prefer logging outside the environment instead)

Reply:
sure